app/config/urls/challenge_subdomain.py

```diff
@@ -26,6 +30,16 @@
     ),
     path("admins/", include("grandchallenge.admins.urls", namespace="admins")),
     path("update/", ChallengeUpdate.as_view(), name="challenge-update"),
+    path(
+        "onboarding-tasks/",
+        OnboardingTaskList.as_view(),
+        name="challenge-onboarding-task-list",
+    ),
+    path(
+        "onboarding-tasks/<pk>/complete",
```

Comment:
Missing type and trailing slash.

```suggestion
        "onboarding-tasks/<uuid:pk>/complete/",
```

Reply:
Thank you. I'll try to be a bit more precise with these. I remember missing them the last changeset as well.